src/main/java/interface_adapter/change_password/ChangePasswordController.java

```diff
@@ -18,8 +18,8 @@ public ChangePasswordController(ChangePasswordInputBoundary userChangePasswordUs
      * @param password the new password
      * @param username the user whose password to change
      */
-    public void execute(String password, String username) {
-        final ChangePasswordInputData changePasswordInputData = new ChangePasswordInputData(username, password);
+    public void execute(String password, String username, String email) {
+        final ChangePasswordInputData changePasswordInputData = new ChangePasswordInputData(username, password, email);
```

Comment:
Good use of clean architecture, packaging the inputdata then passing to interactor